Super heavy weapons (#14)
GameBrains/er-core: modules/special-rules/pages/close-combat-weapons.adoc

---

Where your War Engine unit is armed with {close-combat-weapons}:

* Double its Assault value when it fights against an enemy war engine in close combat.

> * If it wins a close combat against an enemy war engine, it automatically inflicts *catastrophic* damage.
> Roll on the losing war engine's Catastrophic Damage table
> +
> NOTE: This is not a hit, so cannot be prevented by the {save} or {holo-fields} abilities.

Reviewer: In `holo-fields.adoc` we've stated this:

>NOTE: If a war engine uses {close-combat-weapons} and it wins a close combat, then it automatically inflicts catastrophic damage.
This is not a 'hit', so the {holo-fields} ability offers no protection against this.

To have both is rather redundant, and to be consistent implies a third instance at `save.adoc`. So, I propose that we should remove this text from `holo-fields.adoc` and just keep the text you've added here. I think that this actually makes more sense anyway, and I suspect that you'll agree. (I think it all started with an FAQ, so its location was a bit haphazard anyway).

I'd just tweak it a little, mainly for active case (also note that "When" works better than "If" with the imperative "Roll on.."):

```suggestion
* When it wins a close combat against an enemy war engine, it automatically inflicts *catastrophic* damage.
Roll on the losing war engine's Catastrophic Damage table.
+
NOTE: This effect is not a 'hit', so the {save} and {holo-fields} abilities offer no protection against it.
```

Author: OK!
